docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md

```diff
@@ -1 +1 @@
-# DR solution based on cross-region deployment in a single cluster
\ No newline at end of file
+# DR solution based on cross-region deployment in a single cluster
```

Comment:
```suggestion
# DR Solution Based on Cross-Region Deployment in a Single Cluster
```

Reply:
This document is missing in v0.9

Reply:
<img width="1008" alt="image" src="https://github.com/user-attachments/assets/9e3df749-399a-406d-81e7-b09e93326525">

It exists. I think that I touch add one new line. We can fix it altogether anyway.